### PR TITLE
Various minor test fixes

### PR DESCRIPTION
**Pull request by pwithnall**
Accumulated fixes and updates to the tests from the last couple of weeks.

Untested.

### CODE REVIEW

**Comment by pwithnall** [comment]
Test run kicked off at https://openqa.dev.endlessm.com/tests/overview?distri=eos&version=master&build=181022-231135&groupid=1.

File: needles/fbe_factory_dialog.json [hunk 1]
```diff
@@ -12,6 +12,14 @@
       "height": 230,
       "type": "match"
     },
+    {
+      "": "Release version string",
```
Comment (wjt) [inline needles/fbe_factory_dialog.json]: I wrote exactly this patch (± a few pixels) last week, but apparently didn't push it and open the PR :(

File: tests/desktop_chrome.pm [hunk 2]
```diff
@@ -20,9 +20,11 @@ sub run {
     # long) and click the ‘Launch’ button from gnome-software to get to Chrome.
     if (check_screen('gnome_software_chrome_details_installed', 10)) {
         assert_and_click('gnome_software_chrome_details_installed', 'left', 10);
-        send_key('ret');  # skip through the ‘Make Google Chrome the default browser dialogue
     }
 
+    # Skip through the ‘Make Google Chrome the default browser’ dialogue.
+    send_key('ret');
```
Comment (wjt) [inline tests/desktop_chrome.pm]: However, this doesn't seem to work. I think you may be racing between the dialog appearing and the test suite pressing ret?

File: needles/fbe_factory_dialog.json (hunk 1 above)
Comment (pwithnall) [reply needles/fbe_factory_dialog.json]: Sad patch panda.

File: tests/desktop_chrome.pm (hunk 2 above)
Comment (pwithnall) [reply tests/desktop_chrome.pm]: Fixed by adding a sleep and a comment.

**Comment by pwithnall** [comment]
Updated some more encyclopedia screenshots and fixed a few other spurious-failure-modes I encountered on the way. Ready for re-review.

**Comment by wjt** [comment]
LGTM if it works.

**Pull request merged**